draft-ietf-scitt-architecture.md

```diff
@@ -290,26 +298,26 @@ SCITT provides a standardized way to:
 
 ### Promotion of a Software Component by Multiple Entities
 
-A software component (e.g., a library) released by a certain original producer is becoming popular.
-The released software component is accompanied by a statement of authenticity (e.g., a detached signature).
+A software component (e.g., a library or software product) released by a certain original producer is common practice for both open-source and commercial offerings.
```

Comment:
```suggestion
A software component (e.g., a library or software product) released by a legitimate producer is common practice for both open-source and commercial offerings.
```

Reply:
I'm fine with this change, in the context of this use case.

Reply:
I support these enhancements.